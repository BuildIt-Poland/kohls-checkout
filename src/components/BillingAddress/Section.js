import styled from 'styled-components';

import { spacingMedium, spacingLarge } from '../../styles/designTokens';

const Section = styled.section`
  margin-bottom: ${spacingMedium};

  &:last-child {
    margin-bottom: ${spacingLarge};
  }
`;

export default Section;
