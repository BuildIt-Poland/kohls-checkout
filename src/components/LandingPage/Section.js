import styled from 'styled-components';

import { spacingLarge } from '../../styles/designTokens';

const Section = styled.section`
  padding-top: ${spacingLarge};

  &:last-child {
    padding-bottom: 6rem;
  }
`;

export default Section;
