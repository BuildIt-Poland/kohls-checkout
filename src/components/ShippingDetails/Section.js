import styled from 'styled-components';

const Section = styled.section`
  margin: 0 auto;
  width: 100rem;
`;

export default Section;
