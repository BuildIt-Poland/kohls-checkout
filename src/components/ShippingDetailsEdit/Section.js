import styled from 'styled-components';

import { spacingMedium } from '../../styles/designTokens';

const Section = styled.section`
  margin-bottom: ${spacingMedium};
`;

export default Section;
