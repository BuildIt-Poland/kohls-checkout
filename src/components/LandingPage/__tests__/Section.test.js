import React from 'react';
import { create } from 'react-test-renderer';

import Section from '../Section';

describe('COMPONENT - LandingPage Section', () => {
  it('renders correctly', () => {
    const component = create(<Section />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
