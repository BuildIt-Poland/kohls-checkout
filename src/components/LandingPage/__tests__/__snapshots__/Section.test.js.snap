// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT - LandingPage Section renders correctly 1`] = `
.c0 {
  padding-top: 3rem;
}

.c0:last-child {
  padding-bottom: 6rem;
}

<section
  className="c0"
/>
`;
